src/cc/BPFTable.cc

```diff
@@ -31,6 +31,60 @@
 
 namespace ebpf {
 
+BPFTable::BPFTable(const TableDesc& desc)
+  : BPFTableBase<void, void>(desc) {}
+
+StatusTuple BPFTable::get_value(const std::string &key_str, std::string &value_str) {
+  char key[desc.key_size];
+  char value[desc.leaf_size];
+
+  StatusTuple r(0);
+
+  r = this->string_to_key(key_str, key);
+  if (r.code() != 0)
+    return r;
+
+  if(!this->lookup(key, value))
```

Comment:
missing space between if and (...can you try running scripts/style-check.sh to see what other style issues may be introduced here?